### PR TITLE
ListBoxy

### PR DESCRIPTION
**Pull request by mikolajkalwa**

### CODE REVIEW

File: lista2/zadanie5/Form1.cs [hunk 1]
```diff
@@ -48,7 +48,30 @@ private void listBox1_SelectedIndexChanged(object sender, EventArgs e)
...
+
+        private void listBox2_MouseDown(object sender, MouseEventArgs e)
+        {
+            if (e.Button == MouseButtons.Right)
+            {
+                listBox2.SelectedIndex = listBox2.IndexFromPoint(e.Location);
+            }
+        }
+
+        private void usunToolStripMenuItem_Click(object sender, EventArgs e)
```
Comment (wiktorkowalski) [inline lista2/zadanie5/Form1.cs]: i had to

File: lista2/zadanie5/Form1.cs [hunk 2]
```diff
@@ -48,7 +48,30 @@ private void listBox1_SelectedIndexChanged(object sender, EventArgs e)
 
         private void listBox2_SelectedIndexChanged(object sender, EventArgs e)
         {
-            label3.Text = listBox2.SelectedItem.ToString();
+            label3.Text = listBox2.SelectedItem?.ToString() ?? "nic";
```
Comment (wiktorkowalski) [inline lista2/zadanie5/Form1.cs]: wouldn't it be better to use string.Empty or ""?

Comment (mikolajkalwa) [reply lista2/zadanie5/Form1.cs]: imo it doesn't matter

Comment (wiktorkowalski) [reply lista2/zadanie5/Form1.cs]: ¯\_(ツ)_/¯

**Review by wiktorkowalski — APPROVED** [review]

**Review by wiktorkowalski — APPROVED** [review]
merguj pan bo w dupie jesteśmy z tą listą

**Pull request merged**